index.html

```diff
@@ -83,25 +84,23 @@
 
             return timersData;
         }
-        
+
         function updateUrlWithTimers(timers) {
             const encodedJson = encodeURIComponent(JSON.stringify(timers));
             const newUrl = new URL(window.location.href);
             newUrl.searchParams.set("timers", encodedJson);
             history.replaceState(null, "", newUrl.toString());
         }
 
-        const timers = parseTimersFromUrl();
-
-        timers.forEach((timer) => {
+        function createCard(timer) {
```

Comment:
Create the "createCard" function to reuse card creation code. Move the existing card creation code into the "createCard" function, and replace the timers.forEach((timer) => { line with the following code: